pybossa/sched.py

```diff
@@ -43,6 +43,7 @@ class Schedulers(object):
 
     locked = 'locked_scheduler'
     user_pref = 'user_pref_scheduler'
+    task_queue = 'task_queue_scheduler'
```

Comment:
Just a minor thought. Should we consider defining constants for:

```python
default = 'default'
breadth_first = 'breadth_first'
depth_first = 'depth_first'
incremental = 'incremental'
depth_first_all = 'depth_first_all'
```

If so, we could then use these constants in [sched.py](https://github.com/bloomberg/pybossa/blob/b36f549efa3f94d80d11b54b44b8ceb82bf6edf3/pybossa/sched.py) in place of the string values.

Reply:
Thank you Kory! I think defining constraints for scheduler is minor because ultimately it reads from [settings_local.py](https://github.com/bloomberg/pybossa/blob/b36f549efa3f94d80d11b54b44b8ceb82bf6edf3/settings_local.py.tmpl#L181)